blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --v2-feature-carousel-cards-gutter: 16px;
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-width: 48px;
+  --v2-feature-carousel-icon-height: 48px;
+  --v2-feature-carousel-svg-width: 24px;
```

Comment:
If the size is the same for width and height we might just use one variable: `--v2-feature-carousel-svg-size`

Reply:
I did think of kept it different for being modular purpose, anyways changed now